backend/package.json

```diff
@@ -13,7 +13,7 @@
     "mongoose": "^7.4.0",
     "multer": "^1.4.5-lts.1"
   },
-  "name": "backend",
+  "name": "backlend",
```

Comment:
Everything looks good! Just wondering was this change a typo? If so can you change it back please. Thanks